src/ServicePulse.Host/vue/src/components/failedmessages/AllDeletedMessages.vue

```diff
@@ -57,18 +62,12 @@ function loadMessages() {
   return loadPagedMessages(groupId.value, pageNumber.value, "", "", startDate.toISOString(), endDate.toISOString());
 }
 
-async function loadGroupDetails(groupId) {
-  const response = await useFetchFromServiceControl(`archive/groups/id/${groupId}`);
-  const data = await response.json();
+async function loadGroupDetails(groupId: string) {
+  const [, data] = await useTypedFetchFromServiceControl<FailureGroup>(`archive/groups/id/${groupId}`);
```

Comment:
Unfortunately it may be too late, but it would be nice for all the URLs to be typed from `routeLinks`